server/package-lock.json

```diff
@@ -0,0 +1,335 @@
+{
```

Comment:
We are using yarn to support yarn workspaces, this makes sure that shared dependencies between the client and server are correctly installed. Make sure to delete this file once you have setup yarn.